LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2015-2016 Intel Corporation
```

Comment:
@mkuculyma, this LICENSE file explains the conditions of Apache license in general. Please, do not make any changes there.